templates/app/react-native/src/api/AccountAPI.js

```diff
@@ -8,7 +8,7 @@ export const login = ({ username, password }) =>
     method: 'POST',
     url: '/connect/token',
     headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
-    data: `grant_type=password&scope=${oAuthConfig.scope}&username=${username}&password=${password}&client_id=${oAuthConfig.clientId}&client_secret=${oAuthConfig.clientSecret}`,
```

Comment:
We can consider checking the `clientSecret` of `oAuthConfig`.
Add it by `if (oAuthConfig.clientSecret)`